src/test/java/org/mondemand/tests/ClientTest.java

```diff
@@ -922,7 +927,6 @@ public void testInternalProblems() throws Exception {
     contexts.set(client, null);
     client.getContextKeys();
 
-    transports.set(client, null);
```

Comment:
i removed this and below, as it is not really testing a valid scenario in Client.